api/report_server.thrift

```diff
@@ -36,6 +36,13 @@ struct RunData{
 }
 typedef list<RunData> RunDataList
 
+struct RunReportCount{
```

Comment:
I'm thinking on that to give a more generic name for this struct. Later we can use it at file counting too (`getFileCounts`).
```thrift
struct RunDataCount{
  1: i64            id,        // unique id of the data
  2: string         name,      // human readable name of the data
  3: i64            count
}
```